doc/aspice_swe3_4/example/iceoryx_component/test/moduletests/test_example_base_class.cpp

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) 2019 by Robert Bosch GmbH. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "source/example_component/example_base_class.hpp"
+#include "test.hpp"
+
+using namespace ::testing;
+using ::testing::Return;
+
+namespace example
+{
+namespace test
```

Comment:
If it is not referred to elsewhere I prefer anonymous namespaces in a test, i.e. just namespace. The name itself is unimportant, it just serves to avoid linker issues with symbols in other test.cpp files which could define the same global variables. Consider removing the name.

Reply:
We introduced this namespace test for doxygen. With that you can find more easily the tests to the corresponding class in the doxygen documentation.